src/applications/personalization/profile/tests/components/direct-deposit/BankInfoCNP.unit.spec.jsx

```diff
@@ -163,7 +166,7 @@ describe('DirectDepositCNP', () => {
       expect(view.queryByLabelText(/account number/i)).not.to.exist;
     });
     it('should handle adding bank info', async () => {
-      userEvent.click(findSetUpBankInfoButton(view));
+      userEvent.click(await findSetUpBankInfoButton(view));
```

Comment:
this `await` shouldn't be needed since `findSetUpBankInfoButton()` doesn't return a Promise.

Reply:
I was playing around with some code and left that in. Sorry about that. I will fix it